src/parser.rs

```diff
@@ -2695,12 +2695,18 @@ impl<'a> Parser<'a> {
         let index_name = self.parse_object_name()?;
         self.expect_keyword(Keyword::ON)?;
         let table_name = self.parse_object_name()?;
+        let using = if self.expect_keyword(Keyword::USING).is_ok() {
```

Comment:
I think we can avoid `is_ok`: here https://github.com/sqlparser-rs/sqlparser-rs/pull/731